Add an ObjectMapper constructor to MappingJackson2JsonView

candrews opened the pull request:
Instead of having to use the default constructor then calling #setObjectMapper(ObjectMapper), this constructor allows the MappingJackson2JsonView to be created and configured with the desired object mapper in one step.

I have signed and agree to the terms of the SpringSource Individual Contributor License Agreement.

candrews committed:
Add an ObjectMapper constructor to MappingJackson2JsonView

Instead of having to use the default constructor then calling #setObjectMapper(ObjectMapper), this constructor allows the MappingJackson2JsonView to be created and configured with the desired object mapper in one step.

sdeleuze commented:
I think that makes sense, could you modify your commit to add a similar `XmlMapper` constructor to `MappingJackson2XmlView` as well?

candrews committed:
Add an XmlMapper constructor to MappingJackson2XmlView

Instead of having to use the default constructor then calling #setObjectMapper(XmlMapper), this constructor allows the MappingJackson2XmlView to be created and configured with the desired xml mapper in one step.

candrews commented:
@sdeleuze I've updated the PR - thanks for the feedback!

sdeleuze commented:
Merged in master with this polished commit https://github.com/spring-projects/spring-framework/commit/d4c74ad34645119f2f18fc813fe5b9e141560210
